pages/common/git-log.md

```diff
@@ -14,6 +14,10 @@
 
 `git log --oneline`
 
+- Show all commits and branches in a graph format:
+
+`git log --oneline --abbrev-commit --decorate --all --graph`
```

Comment:
Ok so online is kind of useful to make the graph look nice.  It looks like `oneline` also runs the abrev-commit (show short commit hash instead of the 40 character one).